tests/e2e/utils/shared.ts

```diff
@@ -80,10 +80,7 @@ export const skipCtbTour = async (page: Page) => {
  */
 export const clickAndWait = async (page: Page, locator: Locator) => {
   await locator.click();
-
-  if (page.context().browser()?.browserType().name() === 'webkit') {
-    await page.waitForLoadState('networkidle');
-  }
+  await page.waitForLoadState('networkidle');
```

Comment:
do you know why it was originally webkit only?

Reply:
Not sure, maybe @innerdvations knows more?

Reply:
webkit is the only browser that actually requires clickAndWait, it doesn't render instantly like the other browsers so you have to have a slight delay in there (we use waitfor `networkidle`) to allow the render to finish before the next step can proceed

Reply:
I can't say 100%, but I think clickAndWait is necessary for anything that opens a modal

Reply:
but running it for all browsers is fine with me, at worst I think it will add a few milliseconds to test runs